rel-1.0.0/debian/core-deps/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM debian:jessie
 # Install .NET Core dependencies
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
+        ca-certificates \
```

Comment:
The comment above this section with this new dependency is no correct.  This is not a .NET Core dependency.  I have seen this type of situation (wanting to call out/group what the various dependencies are for) handled in a variety of ways within official Dockerfiles

1. Use separate layers for different kinds of dependencies
2. User "variables" to define the different kinds of dependencies and then one apt-get update statement that references the variables.
3.  Use one apt-get update statement with comments separating out the different kind of dependencies.

Each one has some pros and cons.  I personally prefer #3 for this Dockerfile.

RUN apt-get update \
    && apt-get install -y --no-install-recommends \
        ca-certificates \
        \
# .NET Core dependencies
        libc6 \
        libcurl3 \
        libgcc1 \
        libgssapi-krb5-2 \
        libicu52 \
        liblttng-ust0 \
        libssl1.0.0 \
        libstdc++6 \
        libunwind8 \
        libuuid1 \
        zlib1g \
    && rm -rf /var/lib/apt/lists/*